application/src/test/java/org/opentripplanner/street/model/edge/StreetEdgeGeofencingTest.java

```diff
@@ -259,10 +261,15 @@ public void pickupFloatingVehiclesWhenStartedInNoDropOffZone() {
 
       // we return 3 states: one for the speculative renting of a vehicle, but with the information
       // of which networks' no-drop-off zones it started in
-      assertEquals(3, states.length);
+      assertEquals(4, states.length);
```

Comment:
Can you update the comment above?

Reply:
I made this a bit more informative in addition to fixing the number.